docs/pkp/authMethods/authHelpers.md

```diff
@@ -1,7 +1,11 @@
+---
+sidebar_position: 1
+---
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Authentication with PKPs
+# Overview
 
```

Comment:
needs an intro specific about authentication with PKPs, the first sentence currently here talks about Lit Actions which can be confusing for someone coming into this section with no context